privacy/enforcement_test.go

```diff
@@ -19,7 +19,6 @@ func TestAny(t *testing.T) {
 			enforcement: Enforcement{
```

Comment:
This is the most important part of the PR to review to ensure policy enforcement makes sense.